### PR TITLE
Fixup whitespace in source code

### PR DESCRIPTION
**Pull request by hughbe**
This makes the code a lot more readable and consistent - not sure if we want this in the git history but putting it out here anyway

### CODE REVIEW

File: src/bitmap.c [hunk 1]
```diff
@@ -887,7 +887,7 @@ GdipCreateBitmapFromScan0 (int width, int height, int stride, PixelFormat format
 #if WORDS_BIGENDIAN
 		int			i;
 #endif
-        
+
```
Comment (akoeplinger) [inline src/bitmap.c]: in cases like this there should just be an empty line, not one with tabs.

File: src/bitmap.c [hunk 2]
```diff
@@ -1434,8 +1434,8 @@ gdip_pixel_stream_has_next (StreamingState *state)
 {
 	if (state != NULL) {
 		return (state->p >= 0)
-		    || ((state->y < (state->region.Y + state->region.Height))
-                    && (state->x < (state->region.X + state->region.Width)));
+			|| ((state->y < (state->region.Y + state->region.Height))
+					&& (state->x < (state->region.X + state->region.Width)));
```
Comment (akoeplinger) [inline src/bitmap.c]: indentation went wrong here.

File: src/bitmap.c [hunk 3]
```diff
@@ -1470,8 +1470,8 @@ gdip_pixel_stream_get_next (StreamingState *state)
 			state->y++;
 
 			state->scan = (BYTE*)(state->data->scan0)
-			            + state->y * state->data->stride
-			            + state->x;
+						+ state->y * state->data->stride
+						+ state->x;
```
Comment (akoeplinger) [inline src/bitmap.c]: and here

File: src/bitmap.c [hunk 4]
```diff
@@ -1510,8 +1510,8 @@ gdip_pixel_stream_get_next (StreamingState *state)
 			state->y++;
 
 			state->scan = (BYTE*)(state->data->scan0)
-			            + state->y * state->data->stride
-			            + state->x * gdip_get_pixel_format_bpp (state->data->pixel_format) / 8;
+						+ state->y * state->data->stride
+						+ state->x * gdip_get_pixel_format_bpp (state->data->pixel_format) / 8;
```
Comment (akoeplinger) [inline src/bitmap.c]: and here. I'll stop commenting on these, I think we should just revert cases where the line is indented with tabs and spaces are just used for alignment.

File: src/font.c [hunk 5]
```diff
@@ -775,7 +776,7 @@ gdip_get_fontfamily_details (GpFontFamily *family, FontStyle style)
 		cairo_scaled_font_t* scaled_ft;
 		FT_Face face = NULL;
 		cairo_matrix_t matrix1, matrix2;
-	        cairo_font_options_t *options = cairo_font_options_create ();
+			cairo_font_options_t *options = cairo_font_options_create ();
```
Comment (akoeplinger) [inline src/font.c]: one tab too much (there are a few cases in this patch).

File: src/general.c [hunk 6]
```diff
@@ -104,27 +104,27 @@ GetLibgdiplusVersion ()
 GpStatus 
 gdip_get_status (cairo_status_t status)
 {
-        if (status == CAIRO_STATUS_SUCCESS)
-                return Ok;
+	if (status == CAIRO_STATUS_SUCCESS)
+			return Ok;
```
Comment (akoeplinger) [inline src/general.c]: one tab too much.

File: src/gifcodec.c [hunk 7]
```diff
@@ -422,9 +422,9 @@ gdip_load_gif_image (void *stream, GpImage **image, BOOL from_file)
 		si = gif->SavedImages[i];
 		img_desc = &si.ImageDesc;
 		if (img_desc->Top < 0 || img_desc->Height < 0 ||
-		    img_desc->Left < 0 || img_desc->Width < 0 ||
-		    (img_desc->Width + img_desc->Left) > screen_width ||
-		    (img_desc->Height + img_desc->Top) > screen_height) {
+			img_desc->Left < 0 || img_desc->Width < 0 ||
+			(img_desc->Width + img_desc->Left) > screen_width ||
+			(img_desc->Height + img_desc->Top) > screen_height) {
```
Comment (akoeplinger) [inline src/gifcodec.c]: good example of where we should just revert to the previous version with tab + space for alignment. (essentially every change in this file)

File: src/image.c [hunk 8]
```diff
@@ -1040,7 +1040,7 @@ GdipSaveImageToStream (GpImage *image, void *stream, GDIPCONST CLSID *encoderCLS
 	if (!stream || !image)
 		return InvalidParameter;
 
-	return NotImplemented; /* GdipSaveImageToStream - not supported */
```
Comment (akoeplinger) [inline src/image.c]: restore comment

File: src/image.c [hunk 9]
```diff
@@ -2274,10 +2274,10 @@ GpStatus WINGDIPAPI
 GdipLoadImageFromDelegate_linux (GetHeaderDelegate getHeaderFunc,
 								 GetBytesDelegate getBytesFunc,
 								 PutBytesDelegate putBytesFunc,
-                                 SeekDelegate seekFunc,
+								 SeekDelegate seekFunc,
 								 CloseDelegate closeFunc,
 								 SizeDelegate sizeFunc,
-                                 GpImage **image)
+								 GpImage **image)
```
Comment (akoeplinger) [inline src/image.c]: it'd probably make more sense to align with GetHeaderDelegate instead.

**Comment by hughbe** [comment]
I've updated the PR to revert the trailing whitespace changes and the comments you made

**Pull request merged**